apps/docs/pages/appendix/manifest.mdx

```diff
@@ -120,22 +116,13 @@ _Complete manifest.json file with default or examples values :_
 {
   "id": "ReplaceAppName",
   "name": "ReplaceAppName",
-  "url": "http://localhost:3000/",
+  "url": "http://localhost:3000",
   "params": {
-    "dappUrl": "http://localhost:3000/",
-    "nanoApp": "ReplaceAppName",
-    "dappName": "ReplaceAppName",
-    "networks": [
-      {
-        "currency": "ethereum",
-        "chainID": 1,
-        "nodeURL": "..."
-      }
-    ]
+    "exampleQueryParam": "value"
```

Comment:
Sorry we might need to change this too because it's actually not a query param directly but instead it's encoded in the `param` query param
```suggestion
    "exampleParam": "value"
```